packages/thumbnail-grid-block/manifest.json

```diff
@@ -0,0 +1,8 @@
+{
+    "appId": "cle7bmsg70001c1w4wmz5c5j3",
+    "searchFields": [
+        { "type": "fondueRte", "settingId": "title" },
+        { "type": "fondueRte", "settingId": "description" }
+    ],
+    "i18nFields": ["title", "description"]
```

Comment:
Does this work when the fields are inside an array `items`?

Reply:
ah, damm. I removed it and just did `"i18nFields": ["items"]`

Reply:
ah fair enough, thanks for the infos